src/common/ui-components/LanguageSelect/LanguageSelect.module.scss

```diff
@@ -3,7 +3,12 @@
 
 .languageButton {
```

Comment:
Not a huge fan of having menu specific styling here. `LanguageSelect` takes in classname as a prop. Could style the containing div at the menu level instead.

Reply:
wow, and they call me non-agreeable